src/GenerateCommand.php

```diff
@@ -250,7 +250,15 @@ private function writeBarrelFiles(array|Collection $children, string $parent): v
             return;
         }
 
-        $children->each(function ($grandkids, $child) use ($parent) {
+        $convertToCamelIfHyphenated = function ($value) {
+            if (str_contains($value, '-')) {
+                $value = Str::camel($value);
+            }
+
+            return $value;
+        };
```

Comment:
Along with hyphenated filenames, I think underscores might want to be considered too?

```suggestion
        $normalizeToCamelCase = function ($value) {
            return Str::of($value)
                ->replace(['-', '_'], ' ')
                ->camel()
                ->toString();
        };
```